packages/generator-one-app-module/generators/app/templates/intl-root-module/__tests__/components/ModuleContainer.spec.jsx

```diff
@@ -0,0 +1,137 @@
+  });
+  it('switches languages when a new locale is selected', () => {
+    const props = {
+      switchLanguage: jest.fn(),
+      languageData: {
+        locale: 'en-US',
+        greeting: 'hi!',
+      },
+      localeName: 'en-US',
+    };
+    const renderedModule = shallow(<<%=moduleNamePascal%> {...props} />);
+    renderedModule.find('#locale-selector').simulate('change', 'en-CA');
+    expect(props.switchLanguage).toHaveBeenCalledWith('en-CA');
+  });
+
+  describe('childRoutes', () => {
+    it('should return an array of Routes', () => {
+      expect(childRoutes()).toEqual(expect.any(Array));
+      childRoutes().forEach((route) => expect(route.type).toEqual(Route));
+    });
+  });
+
+  describe('mapStateToProps', () => {
+    it('should return the locale name', () => {
+      const mockState = fromJS({
+        intl: { activeLocale: 'en-US' },
+      });
+      expect(mapStateToProps(mockState)).toMatchObject({
+        localeName: 'en-US',
+      });
+    });
+    it('should return an empty object for languageData if there is no language data available', () => {
+      const mockStateWithNoLanguagePack = fromJS({
+        intl: { activeLocale: 'en-US' },
+      });
+      const mockStateWithIncompleteLanguagePack = fromJS({
+        intl: {
+          activeLocale: 'en-US',
+          languagePacks: {
+            'en-US': {
+              '<%=modulePackageName%>': {
+                greeting: 'hello',
+              },
+            },
+          },
+        },
+      });
+      expect(mapStateToProps(mockStateWithNoLanguagePack)).toMatchObject({
+        localeName: 'en-US',
+        languageData: {},
+      });
+      expect(mapStateToProps(mockStateWithIncompleteLanguagePack)).toMatchObject({
+        localeName: 'en-US',
+        languageData: {},
+      });
+    });
+    it('should return languageData if it is available', () => {
+      const mockState = fromJS({
+        intl: {
+          activeLocale: 'en-US',
+          languagePacks: {
+            'en-US': {
+              '<%=modulePackageName%>': {
+                data: {
+                  greeting: 'hello',
+                },
+              },
+            },
+          },
+        },
+      });
+      expect(mapStateToProps(mockState)).toMatchObject({
+        localeName: 'en-US',
+        languageData: {
+          greeting: 'hello',
+        },
+      });
+    });
+  });
+
+  describe('mapDispatchToProps', () => {
+    it('should update the browser locale and then reload the language pack', async () => {
+      const mockDispatch = jest.fn();
+      const { switchLanguage } = mapDispatchToProps(mockDispatch);
+      await switchLanguage({ target: { value: 'en-US' } });
+      expect(mockDispatch).toHaveBeenNthCalledWith(1, 'Switching locale to en-US');
+      expect(mockDispatch).toHaveBeenNthCalledWith(2, 'I am loading the language pack for <%=modulePackageName%> and my fallback locale is en-US');
+    });
+  });
+
+  describe('holcronModule load', () => {
```

Comment:
`holcronModule` should be `holocronModule`. Make sure that this is fixed in every other module template.

Reply:
fixed in intl-child and intl-root template test files